src/SDKs/WebSites/Management.Websites/Properties/AssemblyInfo.cs

```diff
@@ -9,7 +9,7 @@
 [assembly: AssemblyDescription("Provides management functionality for Microsoft Azure Web Sites.")]
 
 [assembly: AssemblyVersion("1.0.0")]
-[assembly: AssemblyFileVersion("1.8.0")]
+[assembly: AssemblyFileVersion("1.9.0")]
```

Comment:
Since you are bumping the major version, both `AssemblyFileVersion` and `AssemblyVersion` must be bumped to `2.0.0.0`

Reply:
will do.